### PR TITLE
Capture stderr

### PR DESCRIPTION
**Pull request by cxa**
Without stderr:
screen shot 2017-07-05 at 10 35 06

With stderr:
screen shot 2017-07-05 at 10 35 17

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,12 +31,15 @@ const runBsbSync = () => {
   execFileSync(bsb, ['-make-world'], { stdio: 'pipe' })
 }
 
-const getBsbErrorMessages = err =>
-  err.match(/File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g)
+const getBsbErrorMessages = err => {
+  const normalErr = /File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g
+  const fatalErr = () => /Fatal error:/g
+  err.match(normalErr) || err.match(fatalErr())
```
Comment (rrdelaney) [inline index.js]: Shouldn't this value be returned?

File: index.js [hunk 2]
```diff
@@ -31,12 +31,15 @@ const runBsbSync = () => {
   execFileSync(bsb, ['-make-world'], { stdio: 'pipe' })
 }
 
-const getBsbErrorMessages = err =>
-  err.match(/File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g)
+const getBsbErrorMessages = err => {
+  const normalErr = /File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g
+  const fatalErr = () => /Fatal error:/g
```
Comment (rrdelaney) [inline index.js]: Why wrap with a function?

**Review by rrdelaney — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (cxa) [inline index.js]: *lazy* here, will not evaluate if `normalErr` is true. Should gain a little performance here I think.

File: index.js (hunk 1 above)
Comment (cxa) [inline index.js]: Ah, my bad, I treated it as OCaml...

**Comment by rrdelaney** [comment]
What's the case that error happens in? I've never seen it before

**Comment by cxa** [comment]
Add an non-exist dir to `sources` bsconfig.json. (In fact I forget to delete it in bsconfig.json after dir deleted)

**Comment by cxa** [comment]
By the way, I think we should not omit the stderr message.

**Review by rrdelaney — APPROVED** [review]

**Comment by rrdelaney** [comment]
Published in [`v1.5.5`](https://github.com/rrdelaney/bs-loader/releases/tag/v1.5.5)

**Pull request merged**